src/menu/menu.service.ts

```diff
@@ -4,17 +4,32 @@ import { RestaurantService } from 'src/restaurant/restaurant.service';
 import { AuthUser } from 'src/common/decorators';
 import { PrismaService } from 'src/prisma/prisma.service';
 import { CreateMenuItemDto, ModificationGroupDto } from './dto';
+import { ConfigService } from '@nestjs/config';
+import { PutObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { v4 as uuidv4 } from 'uuid';
+import { MenuItemImage } from '@prisma/client';
 
 @Injectable()
 export class MenuService {
-  menuId: number;
+  private readonly s3Client = new S3Client({
```

Comment:
let's create s3Service. It should have just simple method like uploadFile(path: string, file: Buffer)